### ISSUE TITLE
[Bug]: wallet tester does not include auxiliary_data_hash and network_id in the transaction body cbor

### ISSUE THREAD

**Issue by dtscalac** [input-output-hk/catalyst-voices#499]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

1. Run wallet tester
2. Connect your wallet
3. Select `Sign transaction` from Wallet actions
4. Enter the following data into the form:

Transaction inputs:
UTXO Hex #1:
```
828258204c1fbc5433ec764164945d736a09dc087d59ff30e64d26d462ff8570cd4be9a70082583900c035332d2dcba35744e880729198459e32c50eb3e179b1fa2247348c80b846ad416f120db94c1a401992950b11b9bc7d65dbb3424c0f8de41b00000002540be400
```

Transaction outputs

| Address #1 | Amount #1 |
| ----------   | ----------  |
|addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw   |   1000000|

Auxiliary Metadata | Transaction Metadata
Text:
| Label #1 | Metadatum #1 |
| ----------   | ----------  |
|1   |   Test|

5. Click `Build` button to get the unsigned transaction bytes
6. Inspect the transaction cbor at cbor.me and inspect if key 7 (auxiliary_data_hash) and key 15 (network_id) are included in the transaction body cbor.
7. Actual result (cbor): 
```
84A300818258204C1FBC5433EC764164945D736A09DC087D59FF30E64D26D462FF8570CD4BE9A700018282581D6082E016828989CD9D809B50D6976D9EFA9BC5B2C1A78D4B3BFA1BB83B1A000F424082583900C035332D2DCBA35744E880729198459E32C50EB3E179B1FA2247348C80B846AD416F120DB94C1A401992950B11B9BC7D65DBB3424C0F8DE41B0000000253FA156B021A00028C55A0F5A1016454657374
```
8. Actual result: Keys 7 & 15 missing
<img width="705" alt="Screenshot 2024-05-16 at 14 33 33" src="https://github.com/input-output-hk/catalyst-voices/assets/166132265/c127fd9e-08eb-4235-a9a5-57d075e14b64">

### Expected Behavior

The transaction body contains the 7 & 15 keys for auxiliary_data_hash and network_id as per [documentation](https://input-output-hk.github.io/catalyst-voices/catalyst-standards/draft-cips/x509-rbac-signing-with-cip30/images/cardano-transaction.svg)

### Project

Frontend

### What operating system are you seeing the problem on?

Other

### What browsers are you seeing the problem on?

Not Applicable

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dtscalac** [comment]
Reopening because of https://github.com/input-output-hk/catalyst-voices/pull/500/files#r1604583882

**Comment by dtscalac** [comment]
Closed in favor of #506
